### ISSUE TITLE
Commands not showing up for me?

### ISSUE THREAD

**Issue by tnrich** [skateshop/sublime-javascript-refactor#8]
The package appears to successfully install in ST3, but no new right click option appears in the file tree, nor does a new command pallet option appear. 

Any help figuring out why that might be would be greatly appreciated. Thanks!

**Comment by tnrich** [comment]
This package maybe just looks half finished?

**Comment by jurassix** [comment]
Yeah I don't think it was ever finished. I'm interested in building out a @Code version if you wanna help?
